package tic_tac_toe;

message Coordinates {
  required int32 x = 1;
  required int32 y = 2;
  required string symbol = 3;
  }

message Good_choice {
  required bool managed_to_put_symbol = 1;
  }
  
message Put_symbol {
  required string symbol = 1;
}